src/pages/home/report/ReportActionItem.js

```diff
@@ -214,6 +214,7 @@ class ReportActionItem extends Component {
                                     hovered
                                     && !this.state.isContextMenuActive
                                     && !this.props.draftMessage
+                                    && this.props.action.pendingAction !== 'delete'
```

Comment:
I don't think we want to disable the entire menu. Just the option to delete or edit. also there is another menu that we will have to make sure does not have these options.

https://user-images.githubusercontent.com/32969087/186720698-4cdb776d-eb16-423b-8d03-4f5d91f15ba7.mp4